tests/xml.js

```diff
@@ -150,6 +152,18 @@ define([
             eq('<output attr=value />', '<output attr="value" />', false);
         });
 
+        it("should fail on escape unquoted attribute value with <", function () {
+            chai.expect(function () {
+                eq('<output value=2<3 />', '<output value="2&lt;3" />', false);
```

Comment:
Ah. I didn't realize this was an already known error. Should there be a comment on why/what we expect this to fail with an error? It looks like eq has two asserts that could error